File: Challenges/fizzbuzz.py
####SOLUTIONS
#1
for x in range(1, 101):
    if x % 3 == 0 and x % 5 == 0:
        print("FizzBuzz")
    elif x % 3 == 0:
        print("Fizz")
    elif x % 5 == 0:
        print("Buzz") 
    else:
        print(x)

#2
a = "Fizz"
b = "Buzz"
for x in range(1, 101):
    print(a+b if x%15 == 0 else a if x%3==0 else b if x%5 == 0 else x)

#3
for x in range(1, 101):
    print("FizzBuzz" if x%3==0 and x%5==0 else "Fizz" if x%3==0 else "Buzz" if x%5==0 else x) 

#4
for x in range(1,101):
 if x%15==0:print("FizzBuzz")
 elif x%3==0:print("Fizz")
 elif x%5==0:print("Buzz")
 else:print(x)

File: Challenges/fizzBuzz.js
for (let i = 1; i <= 100; i++){
    let out = '';
    if (i % 3 == 0) out += 'Fizz';
    if (i % 5 == 0) out += 'Buzz';
    console.log(out || i);
}

function FizzBuzz(aTarget) {
    for (var i = 1; i <= aTarget; i++) {
        var result = "";
        if (i%3 === 0) result += "Fizz";        
        if (i%5 === 0) result += "Buzz";
        if (result.length ===0) result = i;

        console.log(result);
    }
}